BodyweightFitness/Main/HomeViewController.swift

```diff
@@ -8,6 +8,8 @@ class HomeViewController: AbstractViewController {
     
     override func viewDidLoad() {
         super.viewDidLoad()
+
+        self.view.backgroundColor = UIColor.primary()
```

Comment:
This one now extends AbstractViewController - is that still a case?

Reply:
Yes, it belongs to a different view. I'll revert it if it's not the case.